action.yml

```diff
@@ -10,10 +10,10 @@ inputs:
     description: "Directory path to the dockerfile"
     required: true
   ecr_repository:
-    description: "Ecr repository name"
+    description: "Ecr repository name."
```

Comment:
Remove the dot since it isn't a sentence. If you want to use a dot create a proper sentence